Jenkinsfile-test

```diff
@@ -112,6 +112,7 @@ pipeline {
       script {
         sh "docker stop dependencies | true"
         sh "docker stop rules | true"
+        sh "rm -rf tdr-configurations lambda"
```

Comment:
I've got a vague memory of a Jenkins setting that lets you start with a clean folder, but I can't find it. Does it ring any bells? The advantage is that if these steps don't get run (e.g. because Jenkins gets shut down in the middle of a job, or runs out of disk space) then it doesn't break the next job.

Reply:
It rings a bell but I can't find it either. Maybe it's only for the normal pipelines and not the multibranch one like this is. Not sure though.

Reply:
Ooh, this looks it might do the job: https://stackoverflow.com/a/46189946/679924

Reply:
Yep, that's worked :+1: Cheers. I'll close the PR.